src/components/Demo/DemoCreate/DemoCreateStart/index.tsx

```diff
@@ -49,7 +49,11 @@ export const DemoCreateStart: FunctionComponent = () => {
           <h3 className="text-center">Loading funds to your wallet...</h3>
         </div>
       ) : error.length > 0 ? (
-        error
+        <div className="mt-12">
+          <h3 className="text-center">
+            There maybe something wrong with the underlying network, please try again later.
```

Comment:
no button to try again ?

Reply:
a try again button might not be ideal in this case because the underlying getBalance would be down for a period of time ~20mins. (as far as i observed just now)